---
title: AEMaaCS中封鎖的復寫佇列
description: 瞭解如何修正復寫佇列因錯誤而封鎖的Adobe Experience Manager as a Cloud Service問題。
solution: Experience Manager as a Cloud Service
product: Experience Manager as a Cloud Service
applies-to: Experience Manager as a Cloud Service
keywords: KCS、AEMaaCS、AEM、復寫佇列、封鎖、錯誤、封裝例外狀況、java
resolution: Resolution
internal-notes: null
bug: true
article-created-by: Amol Mahajan
article-created-date: 2/2/2024 11:18:45 AM
article-published-by: Amol Mahajan
article-published-date: 2/2/2024 11:46:24 AM
version-number: 3
article-number: KA-23465
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=022354d2-bcc1-ee11-9079-6045bd006704
exl-id: f600b6f4-95ad-482f-b4d7-726899f27f74
source-git-commit: dd19f78d752827e48b7dc68adcd95500f2ffbca0
workflow-type: tm+mt
source-wordcount: '251'
ht-degree: 4%

---

# AEMaaCS中封鎖的復寫佇列


Adobe Experience Manager as a Cloud Service (AEMaaCS)中的復寫佇列有時會遭到封鎖。 依照本文的步驟操作，以透過管理員許可權修正此問題。

## 說明 {#description}


### <b>環境</b>

Adobe Experience Manager as a Cloud Service 



### <b>問題/症狀</b>

其中一個AEMaaCS復寫佇列因為 `javax.jcr.NamespaceException: Unknown namespace prefix: ns22_exception`.



棧疊追蹤：




```
[ EventAdminAsyncThread #3]  org.apache.sling.distribution.journal.impl.publisher.DistributionPublisher [ null]  Message: Failed attempt (37/infinite) to import the distribution package PackageMessage(pubSlingId=d5e89d11-a40d-4b3f-a8f4-ee478c5b91aa, reqType=ADD, pkgId=dstrpck-1698635668289-1c825ebd-f03d-4d56-97b4-790c169facdc, pkgType=journal_filevault, pkgLength=41118, pubAgentName=publish, userId=replication-service, paths=[ /content/dam/product/photography/129/12999/1299989/PS1299989-731_HB.tif] , deepPaths=[ ] ) at offset=74663574 because of 'javax.jcr.RepositoryException: Some errors occurred while installing packages. Please check the logs for details. First exception is logged as cause.', the importer will retry later,
Stacktrace: org.apache.sling.distribution.common.DistributionException: javax.jcr.RepositoryException: Some errors occurred while installing packages. Please check the logs for details. First exception is logged as cause.
at org.apache.sling.distribution.serialization.impl.vlt.FileVaultContentSerializer.importFromStream(FileVaultContentSerializer.java:159)
at org.apache.sling.distribution.packaging.impl.InMemoryDistributionPackageBuilder.installPackageInternal(InMemoryDistributionPackageBuilder.java:119)
at org.apache.sling.distribution.packaging.impl.AbstractDistributionPackageBuilder.installPackage(AbstractDistributionPackageBuilder.java:156)
at org.apache.sling.distribution.monitor.impl.MonitoringDistributionPackageBuilder.installPackage(MonitoringDistributionPackageBuilder.java:92)
at org.apache.sling.distribution.serialization.impl.vlt.VaultDistributionPackageBuilderFactory.installPackage(VaultDistributionPackageBuilderFactory.java:317)
at org.apache.sling.distribution.journal.bookkeeper.PackageHandler.installAddPackage(PackageHandler.java:78)
at org.apache.sling.distribution.journal.bookkeeper.PackageHandler.apply(PackageHandler.java:61)
at org.apache.sling.distribution.journal.bookkeeper.BookKeeper.importPackage(BookKeeper.java:155)
at org.apache.sling.distribution.journal.impl.subscriber.DistributionSubscriber.processQueueItem(DistributionSubscriber.java:396)
at org.apache.sling.distribution.journal.impl.subscriber.DistributionSubscriber.fetchAndProcessQueueItem(DistributionSubscriber.java:348)
at org.apache.sling.distribution.journal.impl.subscriber.DistributionSubscriber.processQueue(DistributionSubscriber.java:326)
at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: javax.jcr.RepositoryException: Some errors occurred while installing packages. Please check the logs for details. First exception is logged as cause.
at org.apache.jackrabbit.vault.fs.io.Importer.run(Importer.java:579)
at org.apache.sling.distribution.serialization.impl.vlt.FileVaultContentSerializer.importFromStream(FileVaultContentSerializer.java:151)
... 11 more
Caused by: org.apache.jackrabbit.vault.packaging.PackageException: Error creating/updating node /content/dam/product/photography/129/12999/1299989/PS1299989-731_HB.tif/jcr:content/metadata
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1177)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:976)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.commit(Importer.java:1018)
at org.apache.jackrabbit.vault.fs.io.Importer.run(Importer.java:531)
... 12 more
Caused by: javax.jcr.NamespaceException: internal error: failed to resolve namespace prefix
at org.apache.jackrabbit.spi.commons.namespace.SessionNamespaceResolver.getURI(SessionNamespaceResolver.java:62)
at org.apache.jackrabbit.spi.commons.conversion.NameParser.parse(NameParser.java:189)
at org.apache.jackrabbit.spi.commons.conversion.ParsingNameResolver.getQName(ParsingNameResolver.java:64)
at org.apache.jackrabbit.spi.commons.conversion.DefaultNamePathResolver.getQName(DefaultNamePathResolver.java:74)
at org.apache.jackrabbit.vault.fs.impl.io.DocViewImporter.updateExistingNode(DocViewImporter.java:1054)
at org.apache.jackrabbit.vault.fs.impl.io.DocViewImporter.addNode(DocViewImporter.java:947)
at org.apache.jackrabbit.vault.fs.impl.io.DocViewImporter.startDocViewNode(DocViewImporter.java:406)
at org.apache.jackrabbit.vault.fs.impl.io.DocViewSAXHandler.startElement(DocViewSAXHandler.java:348)
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.startElement(AbstractSAXParser.java:510)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:374)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2710)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:534)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:888)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:824)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1216)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:635)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:324)
at org.apache.jackrabbit.vault.fs.io.DocViewParser.parse(DocViewParser.java:254)
at org.apache.jackrabbit.vault.fs.impl.io.AbstractArtifactHandler.importDocView(AbstractArtifactHandler.j
```



## 解決方法 {#resolution}


若從Golden-Publish發佈佇列UI中刪除請求，將會解除封鎖佇列。 執行以下指定步驟：



1. 解除封鎖Golden-Publish佇列。 前往 <b>AEM開始</b> `>`  <b>工具</b> `>`  <b>部署</b> `>`  <b>發佈UI</b> 按一下「佇列」表格中具有粗體名稱的列。


2. 在下一頁，識別並記錄封鎖專案（很可能是「資產」）的路徑，然後選取並選擇從佇列中移除該專案。 如此一來，其餘的啟用作業都能繼續進行。


3. 導覽至上一步驟中所記錄路徑中的資產，然後選擇 <b>管理發布</b>. 選取 <b>*取消發佈</b>* 和 <b>*排程：現在</b>*，然後按一下 <b>下一個</b>.


4. 僅選取上的資產 <b>範圍</b> 頁面並按一下 <b>取消發佈</b>. 確認 <b>繼續</b> 已識別任何參照時。


5. 確認資產已取消發佈後，再次導覽至資產並選擇 <b>快速發佈</b>.


6. 確認資產已成功發佈，且散佈佇列未遭到封鎖。
